src/v1/destinations/linkedin_ads/networkHandler.js

```diff
@@ -0,0 +1,106 @@
+const lodash = require('lodash');
+const { TransformerProxyError } = require('../../../v0/util/errorTypes');
+const { prepareProxyRequest, proxyRequest } = require('../../../adapters/network');
+const { isHttpStatusSuccess, getAuthErrCategoryFromStCode } = require('../../../v0/util/index');
+
+const {
+  processAxiosResponse,
+  getDynamicErrorType,
+} = require('../../../adapters/utils/networkUtils');
+const tags = require('../../../v0/util/tags');
+const {
+  constructPartialStatus,
+  createResponseArray,
+} = require('../../../cdk/v2/destinations/linkedin_ads/utils');
+
+// eslint-disable-next-line consistent-return
+const responseHandler = (responseParams) => {
```

Comment:
Can we add error code documentation link ?

Reply:
added